src/OrchardCore.Modules/OrchardCore.Sms.Azure/Views/AzureSmsSettings.Edit.cshtml

```diff
@@ -0,0 +1,29 @@
+@using OrchardCore.Sms.Azure.Services
+@using OrchardCore.Sms.Azure.ViewModels
+@using OrchardCore.Sms
+
+@model AzureSettingsViewModel
+
+<div class="mb-3">
+    <div class="form-check">
+        <input type="checkbox" class="form-check-input" asp-for="IsEnabled" checked="@Model.IsEnabled" role="button" data-bs-toggle="collapse" data-bs-target="#Azure_Provider_Settings" aria-expanded="@Model.IsEnabled.ToString().ToLowerInvariant()" aria-controls="Azure_Provider_Settings" />
+        <label class="form-check-label" asp-for="IsEnabled">@T["Enable Azure Provider"]</label>
```

Comment:
The his should be kept Enable because of how we document the change 
```suggestion
        <label class="form-check-label" asp-for="IsEnabled">@T["Enable"]</label>
```